core/test/lib/network-request-test.js

```diff
@@ -111,10 +111,10 @@ describe('NetworkRequest', () => {
   describe('update fetch stats for Lightrider', () => {
```

Comment:
Add tests for the new timestamps?

Reply:
The test for `_updateTimingsForLightrider` or something else? There aren't really any tests here for startTime, endTime, etc. and I'm not sure what it'd be other than `check _onX() multiplies timestamp by 1000` or whatever.

Reply:
oh, doh, I guess I should just add the other fields here in getRequest()

Reply:
Added - but I had to make a change to `getResponseReceivedEvent` to fix roundtrip record->log errors. Don't understand why creating a timings was behind a flag,  but now it isn't and it fixed the roundtripping problem... seem ok?